eng/pipelines/coreclr/templates/build-and-run-perf-ios-scenarios.yml

```diff
@@ -1,5 +1,5 @@
 parameters:
-  hybridGlobalization: False
+  hybridGlobalization: True
```

Comment:
If `false` is not valid anymore then we can drop this parameter.

Reply:
Yes. Initially I suggested the update after Preview 1 cutoff to avoid any possibility of rollbacks. However, this should be a simple change, so I removed it. There is a similar logic in the `dotnet/performance` repository; the decision is up to @LoopedBard3.

Reply:
Removing the parameter here makes sense. Once we pass the cutoff, I will go back and remove any further down parameters and changes made to flow hybridglobalization for iOS (e.g. there is a parameter for it in the make file, etc.).

Reply:
We need to keep this parameter, as we are planning to support also our ICU libs (that will be implemented in follow up PRs)

Reply:
Reverted.